BUILD.bazel

```diff
@@ -287,26 +296,26 @@ boom_regfile_rams = {
             "floorplan": BLOCK_FLOORPLAN | {
                 # "CORE_UTILIZATION": "200",
                 # "CORE_ASPECT_RATIO": "4",
-                "DIE_AREA": "0 0 400 400",
-                "CORE_AREA": "2 2 298 298",
-                "IO_CONSTRAINTS": "$(location {})".format(io_constraint),
+                "DIE_AREA": ram_data["die_area"],
+                "CORE_AREA": ram_data["core_area"],
+                "IO_CONSTRAINTS": "$(location {})".format(ram_data["io_constraint"]),
             },
             "place": {
                 "PLACE_DENSITY": {
                     "regfile_128x64": "0.42",
                     "regfile_128x65": "0.3",
                 }.get(ram),
-                "IO_CONSTRAINTS": "$(location {})".format(io_constraint),
+                "IO_CONSTRAINTS": "$(location {})".format(ram_data["io_constraint"]),
             },
         },
         stage_sources = {
             "synth": [":constraints-sram"],
-            "floorplan": [io_constraint],
-            "place": [io_constraint],
+            "floorplan": [ram_data["io_constraint"]],
+            "place": [ram_data["io_constraint"]],
         },
         verilog_files = ["mock/" + ram + ".sv"],
```

Comment:
These are mocked register files, i.e. they do not have all the rows. Use "rtl/" for original RTL.

So this leaves me with a question: what size do the new sizes correspond to? The mock/ or rtl/ version?